client/gutenberg/editor/style.scss

```diff
@@ -333,4 +337,6 @@ $gutenberg-theme-toggle: #11a0d2;
 			border-radius: 20px;
 			margin: 0 8px;
 	}
-}
```

Comment:
Since this an innocuous change and is also breaking the CSS/Sass coding guidelines, I'd say we can revert it.